Week3Assinment1_Rajshukla #7.cpp

```diff
@@ -0,0 +1,36 @@
+#include<iostream>
+using namespace std;
+
+    int main()
+    {
+        int arr[10],s,k,i,temp;
+     cout<<"enter array size:";
+      cin>>s;
+     cout<<"enter array elements:";
+      for(i=0;i<s;i++)
+         { cin>>arr[i];}
+
+        for(i=1;i<s;i++)
+     {
+        for(int j=0;j<(s-i);j++)
+            if(arr[j]>arr[j+1])
+               {
+                  temp=arr[j];
+                  arr[j]=arr[j+1];
+                  arr[j+1]=temp;
+               }
+     }
+
+        cout<<"\n enter key:";
+         cin>>k;
+         for(i=0;i<s;i++)
+         {
+      if(k==i)
+            cout<<" found at index:"<<arr[i];

```

Comment:
what would be the output of your program if i gave it 
Input : arr[] = {30, 40, 50, 10, 20}
        key = 10   
	Expected Output : Found at index 3